test/test_helper.rb

```diff
@@ -31,7 +31,10 @@
   config.acceptance_test_framework = :active_support
 end
 
-WebMock.disable_net_connect!(allow_localhost: true)
+WebMock.disable_net_connect!({
+  allow_localhost: true,
+  allow: "chromedriver.storage.googleapis.com",
```

Comment:
We've actually been removing the webdrivers gem everywhere (it breaks things with govuk-docker on M1 macs).

I'd suggest rather than adding this you instead remove this line and the webdrivers gem. If that turns out to be a pain or you're in a hurry I'm happy to approve this and open a follow up PR that does it.

Reply:
Happy to do that, sounds like the right thing to do here. I removed these lines and the webdrivers gem from `Gemfile`, but now getting unable to connect to chromedriver errors. What else needs changing?

Reply:
Thanks for the help @kevindew, have updated as suggested.